tests/install_upgrade_operators/node_component/utils.py

```diff
@@ -61,7 +61,7 @@
 SUBSCRIPTION_TOLERATIONS = [
     {
         "effect": "NoSchedule",
-        "key": MASTER_NODE_LABEL_KEY,
+        "key": f"{NODE_ROLE_KUBERNETES_IO}/master",
```

Comment:
Should be `f"{NODE_ROLE_KUBERNETES_IO}/control-plane"`? 
Maybe we want to have a constant for it?

Reply:
No, it is not possible to use control-plane label here.